pkg/cri/cri.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package containerutils
+package cri
```

Comment:
I don't think `cri` is accurate. The whole point of this package is that each runtime behaves subtly differently and not under the Kubernetes CRI spec.

Something that has less existing meaning would be better I think. Eg. `containerruntimes` or just `container`.

Reply:
Will do! My first attempt was `containerruntime` :+1: